### ISSUE TITLE
Code Formatter Error - Cannot parse when cell contains lines that end with semicolon.

### ISSUE THREAD

**Issue by niwako** [ryantam626/jupyterlab_code_formatter#155]
**Checklist prior to opening an issue**
- [x] I have followed fully the installation steps laid out in the documentation site.
- [x] I have restarted jupyterlab.
- [x] I have read the FAQ section in the documentation site.

**Describe the bug**

Sometimes, a Jupyter notebook contains cells with lines of code that end in a semicolon in order to suppress the output of that line: 

<img width="641" alt="Screen Shot 2020-07-01 at 10 00 58 PM" src="https://user-images.githubusercontent.com/44518134/86318456-cf1edd00-bbe6-11ea-9016-8b33f07bf987.png">

When jupyterlab_code_formatter operates on these cells, it produces an error:

<img width="683" alt="Screen Shot 2020-07-01 at 10 01 11 PM" src="https://user-images.githubusercontent.com/44518134/86318470-d940db80-bbe6-11ea-96b9-c1afd88336d4.png">

**Comment by shadanan** [comment]
And also seems to append an additional semicolon.

**Comment by ryantam626** [comment]
Hello! 
Can you post output of
`pip freeze`
`jupyter labextension list`
`jupyter serverextension list`
please?

I believe you have an outdated version

**Comment by ryantam626** [comment]
I believe I fixed it in version 1.3.0 https://jupyterlab-code-formatter.readthedocs.io/en/latest/changelog.html#id2

**Comment by niwako** [comment]
Hi Ryan. Here's the outputs of the commands you asked for. In the meantime, I'll try updating everything.

```
$ pip3 freeze
altair==4.1.0
appdirs==1.4.4
appnope==0.1.0
attrs==19.3.0
backcall==0.1.0
black==19.10b0
bleach==3.1.5
certifi==2020.4.5.1
chardet==3.0.4
click==7.1.2
colorama==0.4.3
cycler==0.10.0
decorator==4.4.2
defusedxml==0.6.0
entrypoints==0.3
gitdb==4.0.5
GitPython==3.1.2
idna==2.9
importlib-metadata==1.6.0
ipykernel==5.2.1
ipython==7.14.0
ipython-genutils==0.2.0
isort==4.3.21
jedi==0.17.0
Jinja2==2.11.2
joblib==0.15.1
json5==0.9.4
jsonschema==3.2.0
jupyter-client==6.1.3
jupyter-core==4.6.3
jupyterlab==2.1.2
jupyterlab-code-formatter==1.3.1
jupyterlab-server==1.1.4
kiwisolver==1.2.0
MarkupSafe==1.1.1
matplotlib==3.2.1
mistune==0.8.4
mpmath==1.1.0
nbconvert==5.6.1
nbdime==2.0.0
nbformat==5.0.6
notebook==6.0.3
numpy==1.18.4
packaging==20.3
pandas==1.0.3
pandocfilters==1.4.2
parso==0.7.0
pathspec==0.8.0
patsy==0.5.1
pexpect==4.8.0
pickleshare==0.7.5
prometheus-client==0.7.1
prompt-toolkit==3.0.5
ptyprocess==0.6.0
Pygments==2.6.1
pyparsing==2.4.7
pyrsistent==0.16.0
python-dateutil==2.8.1
pytz==2020.1
pyzmq==19.0.1
regex==2020.6.8
requests==2.23.0
scikit-learn==0.23.1
scipy==1.4.1
seaborn==0.10.1
Send2Trash==1.5.0
six==1.14.0
sklearn==0.0
smmap==3.0.4
statsmodels==0.11.1
sympy==1.5.1
terminado==0.8.3
testpath==0.4.4
threadpoolctl==2.1.0
toml==0.10.1
toolz==0.10.0
tornado==6.0.4
traitlets==4.3.3
typed-ast==1.4.1
urllib3==1.25.9
vega-datasets==0.8.0
wcwidth==0.1.9
webencodings==0.5.1
zipp==3.1.0
```

```
$ jupyter labextension list
JupyterLab v2.1.2
Known labextensions:
   app dir: /usr/local/share/jupyter/lab
        @ryantam626/jupyterlab_code_formatter v1.3.1  enabled  OK
        nbdime-jupyterlab v2.0.0  enabled  OK
```

```
$ jupyter serverextension list
config dir: /Users/niwako/.jupyter
    jupyterlab_code_formatter  enabled
    - Validating...
      jupyterlab_code_formatter  OK
    jupyter_nbextensions_configurator  enabled
    - Validating...
Error loading server extension jupyter_nbextensions_configurator
      X is jupyter_nbextensions_configurator importable?
config dir: /usr/local/etc/jupyter
    jupyterlab  enabled
    - Validating...
      jupyterlab 2.1.2 OK
    nbdime  enabled
    - Validating...
      nbdime 2.0.0 OK
```

**Comment by ryantam626** [comment]
Just to confirm, it works with just one single `;` but not two or more `;` right?

**Comment by niwako** [comment]
No, it fails on a single `;` **and** appends an additional `;` to make two of them prior to showing the error message.

**Comment by niwako** [comment]
Oh, sorry, I misunderstood. Yes, it also fails on more than one. Actually, it keeps appending additional semicolons each time you run the formatter.

**Comment by niwako** [comment]
![jupyterlab-black-bug-demo](https://user-images.githubusercontent.com/44518134/86432160-aa406d80-bcab-11ea-96e0-a1249dc142d6.gif)

**Comment by ryantam626** [comment]
Hi sorry for being AFK, I am looking into this now

**Comment by ryantam626** [comment]
I have successfully replicated it, will try to get a fix out soon!

**Comment by ryantam626** [comment]
1.3.2 is now published, and that should have fixed the issue!

**Comment by niwako** [comment]
Hey Ryan. I updated all the packages, and I now have version 1.3.3 of jupyterlab_code_formatter, but the problem persists. Restarted jupyter lab, and the kernel. 

Here's my latest pip freeze, etc:

```
$ pip3 freeze
altair==4.1.0
appdirs==1.4.4
appnope==0.1.0
attrs==19.3.0
backcall==0.1.0
black==19.10b0
bleach==3.1.5
certifi==2020.4.5.1
chardet==3.0.4
click==7.1.2
colorama==0.4.3
cycler==0.10.0
decorator==4.4.2
defusedxml==0.6.0
entrypoints==0.3
gitdb==4.0.5
GitPython==3.1.2
idna==2.9
importlib-metadata==1.6.0
ipykernel==5.2.1
ipython==7.14.0
ipython-genutils==0.2.0
isort==5.0.7
jedi==0.17.0
Jinja2==2.11.2
joblib==0.15.1
json5==0.9.4
jsonschema==3.2.0
jupyter-client==6.1.3
jupyter-core==4.6.3
jupyterlab==2.1.5
jupyterlab-code-formatter==1.3.3
jupyterlab-server==1.1.4
kiwisolver==1.2.0
MarkupSafe==1.1.1
matplotlib==3.2.2
mistune==0.8.4
mpmath==1.1.0
nbconvert==5.6.1
nbdime==2.0.0
nbformat==5.0.6
notebook==6.0.3
numpy==1.19.0
packaging==20.3
pandas==1.0.5
pandocfilters==1.4.2
parso==0.7.0
pathspec==0.8.0
patsy==0.5.1
pexpect==4.8.0
pickleshare==0.7.5
prometheus-client==0.7.1
prompt-toolkit==3.0.5
ptyprocess==0.6.0
Pygments==2.6.1
pyparsing==2.4.7
pyrsistent==0.16.0
python-dateutil==2.8.1
pytz==2020.1
pyzmq==19.0.1
regex==2020.6.8
requests==2.23.0
scikit-learn==0.23.1
scipy==1.5.1
seaborn==0.10.1
Send2Trash==1.5.0
six==1.14.0
sklearn==0.0
smmap==3.0.4
statsmodels==0.11.1
sympy==1.6.1
terminado==0.8.3
testpath==0.4.4
threadpoolctl==2.1.0
toml==0.10.1
toolz==0.10.0
tornado==6.0.4
traitlets==4.3.3
typed-ast==1.4.1
urllib3==1.25.9
vega-datasets==0.8.0
wcwidth==0.1.9
webencodings==0.5.1
zipp==3.1.0
```

```
$ jupyter labextension list
   app dir: /usr/local/share/jupyter/lab
        @ryantam626/jupyterlab_code_formatter v1.3.3  enabled  OK
        nbdime-jupyterlab v2.0.0  enabled  OK
```

```
config dir: /Users/niwako/.jupyter
    jupyterlab_code_formatter  enabled
    - Validating...
      jupyterlab_code_formatter  OK
config dir: /usr/local/etc/jupyter
    jupyterlab  enabled
    - Validating...
      jupyterlab 2.1.5 OK
    nbdime  enabled
    - Validating...
      nbdime 2.0.0 OK
```

**Comment by niwako** [comment]
Looking through the most recent commits, I don't actually see any that addresses the semicolon problem -- did you forget to merge to master?

**Comment by ryantam626** [comment]
Sorry, I made a mistake when rebasing, have now published 1.3.4 which includes the fix published in 1.3.2 (but not 1.3.3).

**Comment by niwako** [comment]
Works now! Thanks Ryan!
